### DISCUSSION TITLE
音标问题!!!

### DISCUSSION THREAD

**Discussion by benabcben** [ZuodaoTech/everyone-can-use-english#939] [Q&A]
生成跟读字幕下面的音标是哪个标注啊,之前记得扫到过说明,现在找不到在哪了,
感觉好多和剑桥词典对不上啊,有人和我有一样的疑问的吗,有点晕了...

**Comment by bluechallenge** [comment]
目前有些单词的音标标注确实与我们习惯的（例如牛津或剑桥词典）有出入，在搞清楚具体原因之前，分享下当下我是怎么处理的，如果有遇到看起来比较别扭的音标，我会：

1. 切换到 Enjoy "Translation" 界面，看 Cambridge Dictionary 是如何标注的，也可以听发音；
2. 如果电脑是Mac的话，选中某个单词后，可以在  "Translation" 界面长按选中某个单词，利用 Mac 内置的 _New Oxford American Dictionary_ 和 _牛津英汉汉英词典_，交叉确认音标；
<img width="1127" alt="image" src="https://github.com/user-attachments/assets/4f7462ec-4595-4b36-9457-d63e6db97493">

3. 当然，选择某个单词后，也可以点击播放键，让 Enjoy 单独播放当前文本里的某个单词，结合音频和音标来确认怎么读更准确；
4. 经过交叉确认后，如果觉得 Enjoy 标注的音标不够准确，可以通过 "Note" 把新的音标标注在单词的下方。
<img width="766" alt="image" src="https://github.com/user-attachments/assets/990ebb9e-7c50-4dd7-b829-57d41ba67a50">

**Reply by benabcben** [reply]
非常感谢,之前不知道可以给每个单词都做笔记
推荐下欧陆词典的鼠标取词,比电脑内置的方便好用
另外请问有微信交流群吗,想有小伙伴一起平时多交流交流
